packages/design-system-react-native/src/primitives/ButtonAnimated/ButtonAnimated.tsx

```diff
@@ -40,7 +40,9 @@ const ButtonAnimated = ({
   };
 
   return (
-    <Animated.View style={scaleStyle}>
+    <Animated.View
+      style={[scaleStyle, { alignItems: 'center', justifyContent: 'center' }]}
```

Comment:
Updating this to make sure the scale animation works as intended

Reply:
_non-blocking:_ You could make this primitive more reusable by moving the scale animation to `ButtonBase`, depending on the values used in `ButtonIcon`.